lib/core/src/lib/datatable/components/datatable/datatable.component.scss

```diff
@@ -11,7 +11,7 @@ $data-table-card-padding: var(--theme-headline-line-height) !default;
 $data-table-cell-top: calc($data-table-card-padding / 2);
 $data-table-thumbnail-width: 65px !default;
 $data-table-cell-min-width: 65px !default;
-$data-table-cell-min-width-no-grow: 100px !default;
+$data-table-cell-min-width-no-grow: 115px !default;
```

Comment:
This will modify the width of all columns and ticket basically needs just specific one to be wider, this can be done in ACA in the config file by using `adf-expand-cell` class in the columns definition object, we don't need to change the default width of all columns